dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,518 @@
 <template>
-  <Card class="col-12">
+  <Card class="col-12 page-wrapper-card">
```

Comment:
After a successfull upload, at two places the small red warnings that this field is mandatory are displayed. I think this should not be the case.

Reply:
done

Reply:
ok